auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -352,24 +300,31 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
     }
 
     this.setAccountChangedHandler(this.setup)
-    this.setup()
+    // this.setup() // not needed since an account change event is forthcoming
   }
 
   private async setup () {
-    this.isLoading = true
+    // ensure syncBusinesses isn't already running
+    if (this.isLoading === 1) {
+      return
+    }
+
+    // ensure syncBusinesses hasn't just been run
+    if (Date.now() - this.lastSyncBusinesses < 2000) {
+      return
```

Comment:
[Vue Debounce Decorator](https://www.npmjs.com/package/vue-debounce-decorator) did not work for `syncBusinesses()` so I implemented this instead.

Reply:
I used a 2-second duration because I found that the account changed handler often took almost 1 second to trigger this (setup method) a second time.

Reply:
Looks good.
Is this load time roughly consistent across environments?

Reply:
I don't actually know. How would I test this? I think it's partly dependent on the environment's resources (web server/db).

That said, I expect it to be about the same. I don't see why it would be much different.

Reply:
Yea, i really don't think you could other than just watch the account change handler in Test (and prod if possible). 
I imagine it would be the same. 
If it was slower by 2-3 seconds for example would that be an issue though?

Reply:
Yes, if it was longer than 2 seconds then the data/table would get loaded twice. That would suck. To test this, we would need a Prod user with an account with many affiliated entities/NRs. I'm thinking we wait for a user to complain about this.

FWIW, as far as I can tell, the delay in Test is the same as in Dev.

Reply:
kk, sounds good!

Reply:
I could bump this up to 3000ms. It just has to be less than the time a speedy user would take to switch between accounts.